### ISSUE TITLE
Change default tolerances for testers

### ISSUE THREAD

**Issue by sethaxen** [JuliaDiff/ChainRulesTestUtils.jl#77]
The default tolerance of `atol=1e-9, rtol=1e-9` for the `isapprox` tests seem like they might be too stringent for 32-bit checks. Perhaps we should have a utility method for settings these tolerances based on the float `eltype` of the inputs?

**Comment by oxinabox** [comment]
yes, in general FiniteDifferences.jl doesn't work well on Float32.
